.github/workflows/test_dev.yml

```diff
@@ -34,8 +34,7 @@ jobs:
           python -m venv env
           source env/bin/activate
           python -m pip install --upgrade pip
-          pip install .
-          pip install -e ./src/openqaoa-core[tests]
+          make dev-install-tests
           pip install ipykernel
```

Comment:
This is perhaps a bit of nitpicking, but I believe ipykernel is already installed as its a dependency of dev-install-tests, no?

Reply:
I don't think we install `ipykernel` for `dev-install-tests`.
https://github.com/entropicalabs/openqaoa/blob/dev_test_pypi/src/openqaoa-core/requirements_test.txt

It might be installed as a dependency for something else maybe. Should we add it in?

Reply:
fair fair, sorry, I may have confused it with ipython